src/vendors/__tests__/__snapshots__/oasisscan.spec.ts.snap

```diff
@@ -158,32 +158,46 @@ Array [
 exports[`oasisscan parse validators 1`] = `
 Array [
   Object {
-    "address": "oasis1qq3xrq0urs8qcffhvmhfhz4p0mu7ewc8rscnlwxe",
-    "current_rate": 0.05,
-    "escrow": 336904789680000000,
-    "media": Object {
-      "email_address": "s@s.f",
-      "logotype": "https://s3.amazonaws.com/s.jpg",
-      "twitter_acc": "https://twitter.com/s",
-      "website_link": "https://s.f",
-    },
-    "name": "s",
-    "rank": 1,
-    "status": "active",
-  },
-  Object {
-    "address": "oasis1qqekv2ymgzmd8j2s2u7g0hhc7e77e654kvwqtjwm",
+    "address": undefined,
     "current_rate": 0.1,
-    "escrow": 326477290610000000,
+    "escrow": 330035031080000000,
     "media": Object {
       "email_address": undefined,
       "logotype": undefined,
       "twitter_acc": undefined,
-      "website_link": "https://www.b.com",
+      "website_link": "https://www.binance.com",
     },
-    "name": "b",
+    "name": "BinanceStaking",
     "rank": 2,
     "status": "active",
   },
+  Object {
+    "address": undefined,
```

Comment:
Something broke

Reply:
ahh I blindly copied `https://api.oasisscan.com/mainnet/validator/list?pageSize=3` response., but that's not what I need in tests. all clear

Reply:
options:
- manually convert fields to snake case
- convert with ValidatorRowFromJSON
- remove `modelPropertyNaming=snake_case` from `openapi-generator-cli`